### ISSUE TITLE
str.join() method does not check argument count

### ISSUE THREAD

**Issue by pylint-bot** [PyCQA/pylint#116]
Originally reported by: **Anonymous**

---

On snippet

```
#!python

print ','.join('1', '2', '3')
```

pylint does not issue any errors, yet, running it with python yields

```
Traceback (most recent call last):
  File "smth.py", line 1, in <module>
    print ','.join('1', '2', '3')
TypeError: join() takes exactly one argument (3 given)
```

---
- Bitbucket: https://bitbucket.org/logilab/pylint/issue/116
